pkg/deb/postinstall

```diff
@@ -1,6 +1,8 @@
 
 mkdir -p /var/log/serviced
-chmod 1777 /var/log/serviced
+chgrp serviced /var/log/serviced
```

Comment:
@smousa - I tested a 5.2.1 install without any of these changes, and the files under `/opt/serviced/var/volumes/tenantID` are owned by a variable number of UID/GID - only some were owned by `root:root`. I also checked a 5.2.2 install with a preliminary set of these changes, and the permissions on `/opt/serviced/var/volumes/tenantID` were varied in the same way. So I don't think having the `serviced-controller` binary with `root:serviced` permissions will have any impact on the ownership of files in the DFS created by various services from their respective containers.